docs/specs/servers/blockchain/blockchain-bundler-api.md

```diff
@@ -0,0 +1,49 @@
+# Blockchain API Bundlers interaction
+
+This API provides a bundlers JSON-RPC interaction. 
+The supported `userOperation` interaction schema is according to v0.7 ERC-4337 specification.
+
+## Make a bundler JSON-RPC request
+
+Used to make a JSON-RPC call to the designated bundler.
+
+`POST /v1/bundler?projectId={projectId}&chainId={chainId}&bundler={bundlerName}`
+
+* `projectId` - Required. The project identifier.
+* `chainId` - CAIP-2 chain identifier.
+* `bundlerName` - Bundler name to use.
+
+The list of currently supported bundler names: 
+* `pimlico` - [Pimlico bundler](https://docs.pimlico.io/permissionless/reference/bundler-actions/sendUserOperation).
+
+#### Request body:
+
+```typescript
+{
+    id:  number, // Unique identifier. Default is 1.
+    jsonrpc: string, // JSON-RPC version. Default 2.0
```

Comment:
I don't feel like we should have defaults for these fields. This looks like standard JSON-RPC and our RPC proxy endpoints don't do this